washingtonpost/elex-clarity
# fix: change client args to work with importer

## Description

This PR fixes an integration error with the Clarity fetcher in elex-data-importer, which calls several methods in the client and expects the county name argument to be `county_name` (not the current `countyname`), as well as to be able to pass additional keyword arguments to `get_current_version` without error. The `county_name` arguments were changed in #16 to accommodate a CLI argument, this renames that argument as it is passed into the CLI using a Click decorator option instead and changes the client arguments back to `county_name`. It also adds an unused `**kwarg` to `get_current_version`. In combination, they fix the error.

## Test Steps

To test this, I checked out `elex-data-importer` and did the following in that repo:
```sh
pip install /Users/zubakskeesc/Desktop/work/elex-clarity # substitute your path to elex-clarity here
npx sls package --stage=dev
npx sls offline start --stage dev
aws stepfunctions --endpoint http://localhost:8083 start-execution --state-machine arn:aws:states:us-east-1:566093341066:stateMachine:ClarityResultsFetcherAndImporter --name $(date +%s) --input '{"date":"2020-11-03","electionId":"105369","state":"GA","level":"precinct","importerParams":{"officeID":"S,H,G","forceUpdate":true},"mode":"live","namespace":"clarity","iterator":{"waitTime":0,"index":0,"limit":1,"done": true},"source":"clarity"}'
```

Without this, the state machine failed with the above errors. With these changes, it succeeded.

To verify this didn't break anything else or the CLI I ran the following in this repo:
```sh
tox
elexclarity 113669 GA --level=precinct --countyName=Appling
```
